### DISCUSSION TITLE
Request for PayPal as an Alternative Payment Method

### DISCUSSION THREAD

**Discussion by CoderBlee** [BlackPythonDevs/blackpythondevs#128] [Ideas]
I would like to propose the addition of PayPal as an alternative payment method for supporting this open-source project. While the current payment methods (such as Patreon, credit cards, or others) are useful, adding PayPal can provide several benefits to the project and its contributors.

PayPal is known for its secure transactions, and many users already have PayPal accounts. Offering this option can make it easier for supporters to donate, subscribe, or contribute without hesitation, as they already trust PayPal.

Some regions have limitations when it comes to using credit cards or other payment gateways. By adding PayPal, you can remove potential barriers for people who want to support the project but don’t have access to the existing payment options.

You can send PayPal donations to : https://www PayPal.com/paypalme/gnomefoundation

For BLACK PYTHON DEVS ONLY, email leadership@blackpythondevs.com for the donation information so we can notify GNOME of the purchase.

**Comment by kjaymiller** [comment]
Because we are a hosted Non-Profit under GNOME, it is possible, albiet not preferred, for supporting Black Python Devs.

You can send paypal donations to - https://www.paypal.com/paypalme/gnomefoundation but you need to leave a comment that says "FOR BLACK PYTHON DEVS ONLY" and email leadership@blackpythondevs.com with the donation information so we can notify GNOME of of the purchase.

**Comment by CoderBlee** [comment]
Okay let me add that, do I add it on the issue or contribution for it?

**Comment by CoderBlee** [comment]
I have updated it on the issue

**Comment by CoderBlee** [comment]
@kjaymiller is the issue correct?

**Comment by kjaymiller** [comment]
@CoderBlee  - I've moved this to a discussion that we can work on and then make an actionable issue.

- [x] I would also be interested in how we can possibly setup a template that would make this process easier for folks who are submitting issues for the first time. 

When filing an issue, it's important to answer some of the important questions. 

- What is the issue and more importantly WHY is it an issue? (You did an excellent job on this!)

Your issue is suggesting that we do a thing but it doesn't mention where.

- Where should this policy be documented? (Internal Documentation/Website)
  -  if you have specifics like filenames you should include them.

Would you like to try again with creating an issue for this?

**Reply by CoderBlee** [reply]
Yes , please I will like to retry again as this my first time raising an issue

**Reply by kjaymiller** [reply]
You're doing a great job!

**Reply by CoderBlee** [reply]
Thank you, I am struggling with adding labels how is that done? is it done after you stated everything and the issue is approved?

**Comment by CoderBlee** [comment]
While the current options such as Patreon, credit cards, and others are useful, including PayPal can provide several benefits for the Black Python Dev project and its contributors.

Why PayPal?

1. Widely Used and Trusted:
   PayPal is widely used for online transactions, and many users already have PayPal accounts. Its reputation for secure payments can make it easier for potential supporters to contribute without hesitation.

2. Overcoming Payment Barriers:
   Some regions or users face limitations with credit cards or other payment gateways. Offering PayPal can help remove potential barriers, enabling those who don’t have access to current payment methods to contribute easily.

3. Increased Flexibility for Donors:  
   With PayPal, we allow donors to choose the payment method they’re most comfortable with. This can encourage more people to donate or subscribe, providing financial support for the development of the project.

Donation Process:

1. PayPal Donations can be made via this link:

-  https://www.paypal.com/paypalme/gnomefoundation

- (https://www.paypal.com/paypalme/gnomefoundation).

2. For Black Python Devs Only:
   - Please email eadership@blackpythondevs.com with the donation details so that we can notify the GNOME Foundation about the contribution.

Since the is no file for it, we can create it, I beleive that this will help in many ways as the Black Python Devs can have events  at schools, universities and in the community etc as bringing people from everywhere to contribute as open source is something broad. If it happens that maybe one the team member their PC is damaged or need to travel , this funds can come in handy. Donations could also help new contributors  with sponsors to keep them going and do many amazing work. 

The policy can be on the website .

Example: "payment-options.md" or "website/donations.html"

Some users may not have access to current payment options like credit cards, and PayPal provides a widely trusted, secure platform that is available globally. 

- Where Should This Be Addressed?

The change should be reflected in the donation section of the website, specifically in 

donations.html.

Add a PayPal link( the link will be available when it is issued and have a file for it) to the donations page.

**Reply by CoderBlee** [reply]
@kjaymiller, is this much better? or still need changes

**Comment by kjaymiller** [comment]
CoderBlee,

This reads as if it were made with AI.

It also shares too much information especially as we've spoken on this in the discussion and come to a resolution.

I would suggest that you reference the discussion here (you can use the number #128) 

and then only provide the actionable changes that need to be made.

**Reply by CoderBlee** [reply]
I did use some AI for better understanding and proper explanation of my ideas. As this is my first time so I honestly don't know much but think outside the box and ask for more assistant from AI

**Reply by CoderBlee** [reply]
by the reference do you mean I should add an adjustment on the 128?

**Reply by kjaymiller** [reply]
> As this is my first time so I honestly don't know much but think outside the box and ask for more assistant from AI

In this case less is more as the more description you add to it the more obscure how to resolve the issue.

Moreover - the description that AI provided talked about services that we don't even use (Patreon)

I would suggest suggest that you make the issue very specific.

Try to follow this model.

title: What should resolving this issue accomplish in one sentence.
description: a short (couple sentences why this is needed and reference any longer discussions that support this)
system information (if applicable): information like Python Version and Operating System
how to replicate (if applicable): steps to replicate the issue
how to fix (if applicable or known)
